.circleci/config.yml

```diff
@@ -36,6 +36,9 @@ jobs:
       - run:
           name: Client tests
           command: $NYC yarn test-client --singleRun
+      - run:
+          name: Setup integration test .percy.yml
+          command: mv .ci.percy.yml .percy.yml
```

Comment:
I'm not crazy about this, but it's needed because adding a `.percy.yml` file breaks the unit tests. 

This can go away once you can pass a custom file path for the `.yml` file. Then the unit tests can have their own mocked & static .yml file.